threat-miner/info.json

```diff
@@ -0,0 +1,278 @@
+                    "status_message": "",
+                    "results": []
+                }
+            ],
+            "parameters": [
+                {
+                    "title": "Imphash",
+                    "type": "text",
+                    "name": "imphash",
+                    "required": true,
+                    "visible": true,
+                    "editable": true,
+                    "value": "1f4f257947c1b713ca7f9bc25f914039",
+                    "placeholder": "e.g: 1f4f257947c1b713ca7f9bc25f914039",
+                    "description": "It requires a hash value to retrieve the malware analysis report.",
+                    "tooltip": "It requires a hash value to retrieve the malware analysis report."
+                },
+                {
+					"title": "Query Type",
+					"type": "select",
+					"name": "query_type",
+					"required": true,
+					"visible": true,
+					"editable": true,
+                    "description": "Select the type of query that is how you want to perform imphash operation.",
+                    "tooltip": "Select the type of query that is how you want to perform imphash operation.",
+					"options": [
+						"Samples"
+					]
+                }
+            ]
+        },
+		{
+            "title": "Get SSDeep",
+            "operation": "get_ssdeep_details",
+            "category": "investigation",
+            "description": "Retrieves the data that detect the level of similarity between two files at the binary level.",
+            "annotation": "ssdeep_details",
+            "output_schema": [
+                {
+                    "status_code": "",
+                    "status_message": "",
+                    "results": []
+                }
+            ],
+            "parameters": [
+                {
+                    "title": "SSDeep",
+                    "type": "text",
+                    "name": "ssdeep",
+                    "required": true,
+                    "visible": true,
+                    "editable": true,
+                    "value": "1536:TJsNrChuG2K6IVOTjWko8a9P6W3OEHBQc4w4:TJs0oG2KSTj3o8a9PFeEHn4l",
+                    "placeholder": "e.g: 1536:TJsNrChuG2K6IVOTjWko8a9P6W3OEHBQc4w4:TJs0oG2KSTj3o8a9PFeEHn4l",
+                    "description": "Provide the SSDeep hash value that attempts to detect the level of similarity between two files at the binary level.",
+                    "tooltip": "Provide the SSDeep hash value that attempts to detect the level of similarity between two files at the binary level."
+                },
+                {
+					"title": "Query Type",
+					"type": "select",
+					"name": "query_type",
+					"required": true,
+					"visible": true,
+					"editable": true,
+                    "description": "Select the type of query that is how you want to perform SSDeep operation.",
+                    "tooltip": "Select the type that is how you want to perform SSDeep operation.",
+					"options": [
+						"Samples"
```

Comment:
Do we need to add Report tagging option as well? and if there is only one option the no need for select type.

Reply:
The Report tagging API is not working. So, removed the select query type field. By default samples query types will be executed.